### ISSUE TITLE
MinerGetBaseInfo return null since since 8 hours ago. 

### ISSUE THREAD

**Issue by s0nik42** [filecoin-project/lotus#5235]
**Describe the bug**

Since 8 hours ago,  MinerGetBaseInfo return null. This was working previously for last 2 months.

I upgrade the miner to v1.4.0 14 hours ago. MinerGetBaseInfo was working after the upgrade. It started returning null 6 hours later.   

**To Reproduce**

Here is the sample request that is not working for me. 
I restar the miner and the daemon without success. 

Request  :

b'{"jsonrpc": "2.0", "method": "Filecoin.MinerGetBaseInfo", "params": ["f010479", 338926, [{"/": "bafy2bzacebzry626yrmzewsapf2qki6ojhcb5edo6beog7czwxl72oeo5k46u"}, {"/": "bafy2bzacecnevfspaxrveynqu2bkvzr4wfo4ilyc5dak4qnrjf4kklq36zx4u"}, {"/": "bafy2bzacedmmi2sdk52333uyl4z3fuzvtdmwaxlzgvyzallyb3lmwaehd7s7o"}, {"/": "bafy2bzacedoyzf4f4frzymxcfn23niwhwmskfvjsp72u5w3ut5czr2r2kqyl6"}, {"/": "bafy2bzacedzjbzml2cxagcq5vxu6nkxfst2dghz65t7mbyftwsorc7utzaxsy"}, {"/": "bafy2bzacecyal5lvwf6hjrb6kmyt33vgsvs6zygxkmzhghdrtwoviv4pfbmfk"}]], "id": 3}'

Answer : 

b'{"jsonrpc":"2.0","result":null,"id":3}\n'

**Version (run `lotus version`):**

lotus version
Daemon:  1.4.0+git.e9989d0e4+api1.0.0
Local: lotus version 1.4.0+git.e9989d0e4

**Additional context**

**Comment by s0nik42** [comment]
We're many facing this problem. the problem didn't solved after I got my power restored. 

I parsed the chain, my miner started getting null result at 337972 so at 6:30AM UTC. so 8hours after my last PoST and more than 6Hours after my upgrade to 1.4.0.  Could you please support us. 

Thank you.

EPOCH / RESULT of MinerGetBaseInfo
337975 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337974 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337974 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337972 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337972 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337971 {'jsonrpc': '2.0', 'result': {'MinerPower': '48724723499008', 'NetworkPower': '1652100683625660416', 'Sectors': [{'SealProof': 3, 'SectorNumber': 391, 'SealedCID': {'/': 'bagboea4b5abcbafpr2l5o6tqb677t4qi6rttw7sbmc6mlkfuextunvnj2yazgilm'}}], 'WorkerKey': 'f3waqhsynu4a3w2a4mcwfaprd4fu6rgd3uuvt7mwj5v24br7u6vcb37xbvgbdgekaav3vcc3m6spz6etuh4aja', 'SectorSize': 34359738368, 'PrevBeaconEntry': {'Round': 433814, 'Data': 'rLEnjwsYsxk1TxucEbT37atJQdGXAVsUD9M2mjswQDbSWSO6ETee6ovwRQsK7gV6F4LuU30dC2FDssigR9CKyvmKjxpF01dspcYP7B4PClM+Tuw98nF/5U87cgNTO7u5'}, 'BeaconEntries': [{'Round': 433815, 'Data': 'reAPCysh4qNjtD4/6YV65xOh/mHjTIsGEr7NqnD7i9/cEq6c4oxBW2lMpVeiYinXB8w5SauVv3srX/s/0tgfXaRs4ZMqWeDPzNdwuCeZ3Su9R9+TzbHCesIXReUSI/z7'}], 'EligibleForMining': False}, 'id': 3}

**Comment by shawnp0wers** [comment]
Just adding that I'm also experiencing this issue. I do not have enough power yet to mine, but the null reply is not the normal response I get. Now, however, it's consistent. Even after a restart of lotus and lotus-miner, plus a reboot.

**Comment by s0nik42** [comment]
Other miner having the same problem starting at exactly the same block !!

f022352
```
337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337972 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337971 {'jsonrpc': '2.0', 'result': {'MinerPower': '73434404978688', 'NetworkPower': '1652100683625660416', 'Sectors': [{'SealProof': 8, 'SectorNumber': 1561, 'SealedCID': {'/': 'bagboea4b5abcbiqnhmmkjjdvxyr4wkszjohlfect2qca35hdtduxxjqmqkob2xik'}}], 'WorkerKey': 'f3uwqu37dt23c5kijeacqgqb3unk364fzprl4zcyjfrow53uwwcy2dhyue423pvxa24xsvlphk7a7awvyfchdq', 'SectorSize': 34359738368, 'PrevBeaconEntry': {'Round': 433814, 'Data': 'rLEnjwsYsxk1TxucEbT37atJQdGXAVsUD9M2mjswQDbSWSO6ETee6ovwRQsK7gV6F4LuU30dC2FDssigR9CKyvmKjxpF01dspcYP7B4PClM+Tuw98nF/5U87cgNTO7u5'}, 'BeaconEntries': [{'Round': 433815, 'Data': 'reAPCysh4qNjtD4/6YV65xOh/mHjTIsGEr7NqnD7i9/cEq6c4oxBW2lMpVeiYinXB8w5SauVv3srX/s/0tgfXaRs4ZMqWeDPzNdwuCeZ3Su9R9+TzbHCesIXReUSI/z7'}], 'EligibleForMining': False}, 'id': 3}
```

**Comment by s0nik42** [comment]
Another one ! f071624

337973 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337972 {'jsonrpc': '2.0', 'result': None, 'id': 3}
337971 {'jsonrpc': '2.0', 'result': {'MinerPower': '5188320493568', 'NetworkPower': '1652100683625660416', 'Sectors': [{'SealProof': 3, 'SectorNumber': 26, 'SealedCID': {'/': 'bagboea4b5abcbxjolmkq3pivubhx6awmvpzg7e5e7t6cnwziuxqbvderun22hqkf'}}], 'WorkerKey': 'f3vewwfwaf2paiygmlf5pfyr6y26ot6h2q3kfnqjtj6mpqhbo4pxdjsral6xnxa4lcob6slnv254vocymreq6a', 'SectorSize': 34359738368, 'PrevBeaconEntry': {'Round': 433814, 'Data': 'rLEnjwsYsxk1TxucEbT37atJQdGXAVsUD9M2mjswQDbSWSO6ETee6ovwRQsK7gV6F4LuU30dC2FDssigR9CKyvmKjxpF01dspcYP7B4PClM+Tuw98nF/5U87cgNTO7u5'}, 'BeaconEntries': [{'Round': 433815, 'Data': 'reAPCysh4qNjtD4/6YV65xOh/mHjTIsGEr7NqnD7i9/cEq6c4oxBW2lMpVeiYinXB8w5SauVv3srX/s/0tgfXaRs4ZMqWeDPzNdwuCeZ3Su9R9+TzbHCesIXReUSI/z7'}], 'EligibleForMining': False}, 'id': 3}

**Comment by s0nik42** [comment]
@magik6k could you solve this in a new release ? This call contains precious information for software using API calls like Farcaster :) thank you !

**Comment by gradoj** [comment]
as a new miner with no power i was seeing MinerGetBaseInfo returning null. it appears to start working some time after pledging a sector

**Comment by shawnp0wers** [comment]
Yeah, the problem is, when you miss wdpost and your power drops to zero, you get null until 7.5 hours AFTER your next successful wdpost. It's a long time with no data, much of which isn't specific to your ability to mine.

**Comment by SBudo** [comment]
Same problem here...
